src/irs/core/s3.py

```diff
@@ -49,52 +49,31 @@ def calc_part_ranges(
 
 
 @dataclass
-class S3IDs:
+class S3Ids:
```

Comment:
As mentioned above, maybe use a naming that is free of S3. E.g. `ObjectAddress`.

Reply:
This is now called StorageIds in 35dd058